src/metabase/lib/filter.cljc

```diff
@@ -172,107 +172,151 @@
     stage-number :- [:maybe :int]]
    (clojure.core/not-empty (:filters (lib.util/query-stage query (clojure.core/or stage-number -1))))))
 
-(defmethod lib.metadata.calculation/display-name-method :mbql.filter/operator
+(defmethod lib.metadata.calculation/display-name-method :operator/filter
   [_query _stage-number {:keys [display-name]} _display-name-style]
   display-name)
 
-(defmethod lib.metadata.calculation/display-info-method :mbql.filter/operator
+(defmethod lib.metadata.calculation/display-info-method :operator/filter
   [_query _stage-number {:keys [display-name] short-name :short}]
   {:short-name (u/qualified-name short-name)
    :display-name display-name})
 
+(defn operator-def
```

Comment:
Unrelated, but I moved this stuff into a separate function because there were a lot of duplicated definitions, not just in this file but in joins as well, and I wanted to de-duplicate.